specification.md

```diff
@@ -1,563 +1,3 @@
 # JSON Payment Protocol Specification V2
 
-Revision 0.1
```

Comment:
We can just delete this file, unless there's some benefit to keeping it around.

Reply:
done